.semaphore/semaphore-runner.sh

```diff
@@ -54,6 +54,10 @@ apt-get install -y fdisk tree libfdisk-dev libp11-kit-dev libssl-dev libpwqualit
 apt-get purge --auto-remove -y unattended-upgrades
 systemctl unmask systemd-networkd
 systemctl enable systemd-networkd
+
+# https://github.com/systemd/systemd/issues/23415
+apt-get install -y accountsservice
+systemctl disable accounts-daemon
 EOF
```

Comment:
This doesn't make sense.

Reply:
Could you be more specific? If there is a better way to address https://github.com/systemd/systemd/issues/23415 I'd appreciate it if you could point me in the right direction

Reply:
Anyway I have no idea what exactly doesn't make sense. If by analogy with 8821518b00453559 the test was supposed to be turned off entirely I don't think that's the right approach in general. Minor packaging issues shouldn't prevent systemd from being tested. If it was supposed to mean that it shouldn't be here in the first place I agree. The package should be fixed so as not bug upstream systemd contributors with this kind of irrelevant stuff. It's not the Debian CI after all.

Looking at https://salsa.debian.org/systemd-team/systemd/-/commit/5738b62f5544d040550a018dcd02701bac4feec8 my best guess would be that stuff is turned off downstream and it should go there. I have no idea.

Reply:
Looking at https://salsa.debian.org/systemd-team/systemd/-/commit/5738b62f5544d040550a018dcd02701bac4feec8 a bit closer it appears it's expected that the tests are run on bullseye so I can't seem to just switch to bookworm here. I'm not going to even try to touch that so I'll go ahead and close the PR. I'm pretty sure it isn't going to be merged anyway :-)

Reply:
I have no idea about the workaround, but the basic concept of this PR looks good to me, and I think we should definitely switch to bookworm.

Reply:
No, i want bullseye covered until it becomes oldstable next year

Reply:
> No, i want bullseye covered until it becomes oldstable next year

I wonder why that would be useful upstream?